docs/skopeo-generate-sigstore-key.1.md

```diff
@@ -0,0 +1,43 @@
+% skopeo-generate-sigstore-key(1)
+
+## NAME
+skopeo\-generate-sigstore-key - Generate a sigstore public/private key pair.
+
+## SYNOPSIS
+**skopeo generate-sigstore-key** [*options*] **--output-prefix** _prefix_
+
+## DESCRIPTION
+
+Generates a public/private key pair suitable for creating sigstore image signatures.
+The private key is encrypted with a passphrase;
+if one is not provided using an option, this command prompts for it interactively.
+
+The private key is written to _prefix_**.private** .
+The private key is written to _prefix_**.pub** .
+
+## OPTIONS
+
+**--output=prefix** _prefix_
```

Comment:
```suggestion
**--output-prefix=prefix** _prefix_
```

Reply:
Thank you! Fixed I think, not quite as suggested.